DependencyInjection/SensioFrameworkExtraExtension.php

```diff
@@ -32,6 +32,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
 
+        if (class_exists('Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface')) {
+            $loader->load('psr7.xml');
+        }
```

Comment:
I think it'd be more natural if we add this as configuration (i.e. maybe I'm using the Psr bridge for middleware, but don't actually want the listeners inside the framework). Something like:

```yml
sensio_framework_extra:
    psr_message:
        enabled: true
```

Or something similar. This isn't quite like the rest of the [config](http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html#configuration) in this bundle, but I find that config kind of awkward.